ui/desktop/electron-app/src/index.js

```diff
@@ -61,7 +62,7 @@ fixPath();
 // Setup logger
 log.initialize();
 log.transports.console.level = false;
-log.transports.file.level = process.env.BOUNDARY_DESKTOP_LOG_LEVEL ?? 'info';
```

Comment:
clarifying question: We would no longer be able to set the log level with this env variable now since we can do it via the settings page?

Reply:
Correct, this will remove the need for using the env variable